packages/@unimodules/core/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### 🎉 New features
 
+- App delegates now can receive the completion handler via the `application:handleEventsForBackgroundURLSession:completionHandler:` method on the iOS. ([#8599](https://github.com/expo/expo/pull/8599) by [@lukmccall](https://github.com/lukmccall))
```

Comment:
- `can now` sounds better than `now can` but they both are correct, another good choice would be `Now app delegates can`
- by the way I think it's more common to use `iOS`/`Android` without `the` but I can't cite the rule 😉 

```suggestion
- App delegates can now handle background URL session events via `application:handleEventsForBackgroundURLSession:completionHandler:` method on iOS. ([#8599](https://github.com/expo/expo/pull/8599) by [@lukmccall](https://github.com/lukmccall))
```